ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -160,30 +181,38 @@ done
 	case "$ARCHITECTURE" in
 		"S390X" )
 			export MACHINE="s390-ccw-virtio"
-			export DRIVE="-drive file=$workFolder/${ARCHITECTURE}.dsk,if=none,id=hd0 -device virtio-blk-ccw,drive=hd0,id=virtio-disk0"
+			export DRIVE="-drive file=$workFolder/$OS.${ARCHITECTURE}.dsk,if=none,id=hd0 -device virtio-blk-ccw,drive=hd0,id=virtio-disk0"
 			export QEMUARCH="s390x"
 			export SSH_CMD="-net user,hostfwd=tcp::$PORTNO-:22 -net nic";;
 		"PPC64LE" )
 			export MACHINE="pseries-2.12"
-			export DRIVE="-hda $workFolder/${ARCHITECTURE}.dsk"
+			export DRIVE="-hda $workFolder/$OS.${ARCHITECTURE}.dsk"
```

Comment:
For syntax consistency
```suggestion
			export DRIVE="-hda $workFolder/${OS}.${ARCHITECTURE}.dsk"
```